libs/vscode-ui/components/src/lib/field/field.component.ts

```diff
@@ -46,9 +47,6 @@ export class FieldComponent implements ControlValueAccessor, OnDestroy {
   valueChangeSub: Subscription;
   OptionComponent = OptionComponent;
 
-  control = new FormControl('');
```

Comment:
finally removing all of this new FormControl down in these components and just connecting thru the ControlContainer 😸